### ISSUE TITLE
create variable "rights" or "copyright" and map Zotero's "Rights" field

### ISSUE THREAD

**Issue by adam3smith** [citation-style-language/zotero-bits#63]
https://forums.zotero.org/discussion/27438?page=1#Item_2
Use case isn't exactly a citation style, but would be very useful in creating quasi-citation labels for images and other artwork

**Comment by swifterslb** [comment]
Sorry to revive an old thread, but I'd like to ask if this has been implemented. Has there been a CSL field that maps "Rights" in Zotero (maybe also "Legal Status" in Mendeley)? My organization would like to mention CC licensing in reference items. Thanks.

**Comment by rmzelle** [comment]
@swifterslb, no. There hasn't been an update to CSL since this ticket was opened. I don't expect a lot of opposition to addition a CSL variable for licensing information, but it will be a while before we have another release ready we can add it to.

**Comment by denismaier** [comment]
So, should we add `license`?

**Comment by dstillman** [comment]
Should we rename Rights to License in Zotero and map it? "Rights" has always been a fairly confusing label.

**Comment by adam3smith** [comment]
I like that, yes.

**Comment by bwiernik** [comment]
Agreed
